tests/e2e/e2e_setup_test.go

```diff
@@ -275,6 +275,7 @@ func (s *IntegrationTestSuite) initGenesis() {
 	oracleGenState.Params.HistoricStampPeriod = 5
 	oracleGenState.Params.MaximumPriceStamps = 4
 	oracleGenState.Params.MedianStampPeriod = 20
+	oracleGenState.Params.MaximumMedianStamps = 2
```

Comment:
I think we should confirm the test still passes with more median stamps (maybe 5) unless that requires rewriting things beyond this parameter

Reply:
It will pass regardless of this number since we look for the median at a specific block height. I was going to put some stuff at the bottom checking that they were pruned correctly but I'd need to add another keeper method. I'll set it to 5 just for funzies.

Reply:
I'll make another PR for this next.